docs/argo-server.md

```diff
@@ -95,7 +103,7 @@ argo-server   LoadBalancer   10.43.43.130   172.18.0.2    2746:30008/TCP   18h
 
 You can get ingress working as follows:
 
-Add `ARGO_BASE_HREF` as environment variable to `deployment/argo-server`.
+Add `ARGO_BASE_HREF` and `ARGO_BASE_HREF` as environment variable to `deployment/argo-server`.
```

Comment:
typo
```suggestion
Add `ARGO_BASE_HREF` and `ARGO_ROOT_PATH` as environment variable to `deployment/argo-server`.
```

Reply:
Thank you for spotting that mistake! I’ve updated the docs based on your suggestion.